libreoffice-core/oox/source/export/vmlexport.cxx

```diff
@@ -1033,13 +1033,6 @@ void VMLExport::Commit( EscherPropertyContainer& rProps, const tools::Rectangle&
                 break;
             default:
 #if OSL_DEBUG_LEVEL > 0
```

Comment:
Note that this won't be included in release builds, so maybe no need to remove it?

Reply:
But either way, no big deal. If this is good to go, feel free to merge.